lib/tooling_invoker/representer_job.rb

```diff
@@ -20,8 +20,13 @@ def tool
       "#{language}-representer"
     end
 
-    def parsed_result
-      result
+    def output
+      {
+        "representation.txt": output_files["representation.txt"],
+        "mapping.json": JSON.parse(output_files["mapping.json"])
+      }
+    rescue StandardError
+      {}
```

Comment:
I feel like we've got a bit of indirection going on here now, which took me a bit of time to get my head around. Having the method setting output_files on job, then them being converted in each job via this.

How about instead we have a writer method on each of these jobs (called `output=`) which takes the hash of filenames and raw files, and then mutates them before storing them? So we don't store them as `output_files`, just as `@output`, and then we can have `attr_reader :output` on the job.rb?

Reply:
I like that. I re-used the indirection that was already there, but the above seems like a better approach.